doc/contributors.rst

```diff
@@ -14,6 +14,7 @@ A (hopefully) updated list of people who contributed to Fatiando a Terra:
 * `Martin Bentley`_ - Nelson Mandela Metropolitan University, South Africa and AEON, South Africa
 * `Victor Almeida`_ - UERJ, Brazil.
 * `M. Andy Kass`_ - United States Geological Survey, USA
+* `Piotr Kurnik`_ - United Kingdom
```

Comment:
Do you want to link your name to some page (a personal page, github profile, etc)? If so, add the link to the lines below like the other contributors. If not, you can remove the back tics and the underline.

Sorry that I didn't see this before.